community/postsrsd/postsrsd.initd

```diff
@@ -0,0 +1,29 @@
+#!/sbin/openrc-run
+
+description="Postfix SRS Daemon"
+pidfile="/run/postsrsd.pid"
+command="/usr/sbin/postsrsd"
+command_args="-D -d ${SRS_DOMAIN} -s ${SRS_SECRET} -f ${SRS_FORWARD_PORT}
+              -r ${SRS_REVERSE_PORT} -p \"${pidfile}\" -u ${RUN_AS}
+              -c \"${CHROOT}\" -a \"${SRS_SEPARATOR}\""
```

Comment:
@Ikke I think what @jirutka means is if you comment one of the options in the confd file the initd will surely fail because it will apply an option without a value. You can use "Parameter Substitution" to solve this and let the confd file to override it.

A user should never have to edit the init.d file and should only edit the confd file. So if some of the options should be able to be removed you will have to add the logic to the initd file depending if the option is set or not.